scripts/SMD_chip_package_rlc-etc/SMD_chip_package_rlc-etc.py

```diff
@@ -60,17 +60,22 @@ def calcPadDetails(self, device_params, ipc_data, ipc_round_base, footprint_grou
         width_tolerance = terminal_width_max - terminal_width_min
 
         if 'terminator_spacing_max' in device_params:
-            spacing_tolerance = device_params['terminator_spacing_max']-device_params['terminator_spacing_min']
-            Gmin = device_params['terminator_spacing_max'] - 2*ipc_data['heel'] - math.sqrt(spacing_tolerance**2 + F**2 + P**2)
+            Tmin = (device_params['body_length_max'] - device_params['terminator_spacing_max'])/2
+            Tmax = (device_params['body_length_min'] - device_params['terminator_spacing_min'])/2
+            Ttol = (Tmax-Tmin)/2
+            Stol_RMS = math.sqrt(Ltol**2+2*(Ttol**2))
+            Smax_RMS = device_params['terminator_spacing_min'] + Stol_RMS
+            Gmin = Smax_RMS - 2*ipc_data['heel'] - math.sqrt(Stol_RMS**2 + F**2 + P**2)
+
         else:
-            terminal_tolerance = device_params['terminal_length_max'] - device_params['terminal_length_min']
-            spacing_tolerance = math.sqrt(length_tolerance**2+terminal_tolerance**2)
+            Ttol = (device_params['terminal_length_max'] - device_params['terminal_length_min'])/2
```

Comment:
I suspect that Ttol should not be divided by 2 here. (or above on line 65)  It doesn't look like you are measuring from the center of the component.  These are absolute lengths

Reply:
Yes it needs to be devided by two. The reason is that the tolerance is relative to the nominal size which is assumed to be between max and min.

Reply:
Ok i re read the relevant parts of the standard. I think you might be right.